dist/test_tag.py

```diff
@@ -55,16 +70,17 @@
 ##### RETRIEVE VALID TAGS #####
 validation_file = open("test_tags.ok", "r")
 
-# The file has the following pattern
-# <COMPONENT>:<TESTING_TYPE>:<TESTING_AREA>:<DESCRIPTION>
-# A tag is made of the three first values: COMPONENT, TEST_TYPE and TESTING_AREA
+# The validation file contains a list of valid tag words.
```

Comment:
Should we say it's one per line ? That's the format expected.

Reply:
bbdb81c9c